tests_v2/test_terminal_multi_image_drop.py

```
@@ -0,0 +1,108 @@
+
+        print(f"PASS: local image drop materialized {len(materialized_paths)} images")
+        return 0
+    finally:
+        if workspace_id:
+            with cmux(SOCKET_PATH) as client:
+                client.close_workspace(workspace_id)
+        for path in materialized_paths:
+            try:
+                Path(path).unlink()
+            except FileNotFoundError:
+                pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(FileNotFoundError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(FileNotFoundError): ...`

(SIM105)